jsapp/js/components/drawer.es6

```diff
@@ -180,16 +180,6 @@ const Drawer = observer(class Drawer extends Reflux.Component {
           { sessionStore.isLoggedIn &&
             <HelpBubble/>
           }
-          { sessionStore.isLoggedIn &&
-            sessionStore.currentAccount.projects_url &&
-            <a href={sessionStore.currentAccount.projects_url}
-              className='k-drawer__link'
-              target='_blank'
-              data-tip={t('Projects (legacy)')}
-            >
-              <i className='k-icon k-icon-globe' />
```

Comment:
Please delete that `globe.svg` from repo - just make sure to leave the `globe-alt.svg` 😉